yt_dlp/extractor/loom.py

```diff
@@ -0,0 +1,103 @@
+from .common import InfoExtractor
+import json
+import urllib.request
+from datetime import datetime
+
+'''
+This scraper was made really fast without really following best practices.
+The webpage string has json data inside it and it would be better if the
+video data was grabbed from there instead of using regex.
+Because loom could change their video requesting api at any time, I decided
+not to work too much on this. If you want to make this scraper better, feel free to do so.
+'''
```

Comment:
ok will do this at the end. 
But it would be better comments were allowed. Would be so much easier to maintain the code imo but whatever

Reply:
Comments are useful (and allowed) when it's not obvious
- what the code is doing,
- why it is doing it,
- or where a hardcoded value comes from.

Otherwise you should strive to have your code speak for itself